ui/src/pageLayout/constants/navigationHierarchy.ts

```diff
@@ -86,30 +114,87 @@ export const generateNavItems = (orgID: string): NavItem[] => {
       icon: IconFont.UsersTrio,
       label: 'Organization',
       shortLabel: 'Org',
-      link: CLOUD
-        ? `${CLOUD_URL}/organizations/${orgID}${CLOUD_USERS_PATH}`
-        : `${orgPrefix}/members`,
+      link: {
+        type: 'link',
+        location: `${orgPrefix}/members`,
+      },
+      cloudExclude: true,
       activeKeywords: ['members', 'about'],
       menu: [
         {
           id: 'members',
           testID: 'nav-subitem-members',
           label: 'Members',
-          link: `${orgPrefix}/members`,
-          cloudExclude: true,
+          featureFlag: 'multiUser',
+          featureFlagValue: false,
+          link: {
+            type: 'link',
+            location: `${orgPrefix}/members`,
+          },
+        },
```

Comment:
This subnavitem only ever renders in OSS.

I think we shouldn't feature flag this subnav item and always link `Members` to `/members` in OSS. That means removing the following subnavitem going to `${CLOUD_URL}/organizations/${orgID}${CLOUD_USERS_PATH}`. Technically OSS has its own MultiUser experience, and we should always render that. When the time comes to ship a new experience in OSS, we can rely on a feature flag that the end user can change as the operator